update_blocklist.nix

```
@@ -40,16 +40,44 @@ in
   # Create an ip set and add each ip to it one by one
 
   # IPv4 and IPv6 regex patterns with CIDR notation support - WARNING: might not be correct for all IPs (e.g. ignore valid ones or accept wrong ones), but seems to work fine
+  # Also supports hosts files, as well as plain lists of domains.
   ipv4_regex="^([0-9]{1,3}\.){3}[0-9]{1,3}(\/[0-9]{1,2})?$"
   ipv6_regex="^([0-9a-fA-F:]+::?[0-9a-fA-F]*)+(\/[0-9]{1,3})?$"
+  domain_regex"^([A-Za-z0-9-]+(?<!-)\.)+[A-Za-z0-9-]+$"
+  host_regex"$^0\.0\.0\.0 $\{domain_regex:1\}"
```

This has to be changed, too, right?
```suggestion
  host_regex"$^0\.0\.0\.0 ''${domain_regex:1}"
```

---

Yeah, oops!